polaris-ui/src/app/features/cases/hooks/use-case-details-state/useCaseDetailsState.ts

```diff
@@ -653,9 +391,16 @@ export const useCaseDetailsState = (
 
       dispatch({
         type: "REGISTER_NOTIFIABLE_EVENT",
-        payload: { documentId, notificationType: "Reclassified" },
+        payload: { documentId, reason: "Reclassified" },
       });
 
+      if (wasDocumentRenamed) {
```

Comment:
is this by passing the polling , and adding the notification event directly based on the user action? Should this be automatically picked up by the refresh pipeline triggered after rename and reclassify

Reply:
as i understand this is needed as we are not showing notifications created as part of user actions in the notification panel.